src/Etsy/RequestValidator.php

```diff
@@ -98,18 +99,14 @@ public static function validateData($args, $methodInfo)
 								$arg = @$arg[0];
 							} else {
 								$item_type = self::transformValueType(@gettype($arg[0]));
+								if($item_type == 'string' && preg_match("/^[\s0-9,]+$/", $arg[0])) {  //is comma separated integer string
```

Comment:
I know that [`array(string|int)` types validation is very weak](https://github.com/kievins/etsy-php/blob/192633faa69304a53b219b895678fa8052a9adae/src/Etsy/RequestValidator.php#L119), but without your change, wasn't it working for `array("1, 2, 3")`? Or you are adding this extra validation to improve it for `array(int)` cases?